plugins/orion/src/components/notification/Notification.tsx

```diff
@@ -8,11 +8,11 @@ import {
 import { Grid } from '@material-ui/core';
 import { PageHeader } from '../PageHeader';
 import { NotificationList } from './NotificationList';
+import { ParodosPage } from '../ParodosPage';
 
 export const Notification = () => {
   return (
-    <Page themeId="tool">
-      <PageHeader />
+    <ParodosPage>
       <Content>
```

Comment:
The `<Content>` is aleady part of the `ParodosPage`. We can wrap children in a fragment `<></>`

Reply:
fixed, please review again